integ-test-promote-template.yml

```diff
@@ -55,6 +55,27 @@ jobs:
           displayName: Install MATLAB
           inputs:
             release: R2023a
+        - task: MathWorks.matlab-azure-devops-extension-dev.RunMATLABCommand.RunMATLABCommand@${{ version }}
+          displayName: Check matlab release
+          inputs:
+            command: version
```

Comment:
Should this be asserting the version? If not, maybe "Print MATLAB version"?

Reply:
Do we ever check that when you request a release, you actually get that release? I think that's what this test was attempting to achieve previously.